### PR TITLE
fcitx5-material-color: update to 20201124

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

fcitx5-material-color: update to 20201124

Package(s) Affected
-------------------

fcitx5-material-color 20201124

Security Update?
----------------

No

Architectural Progress
----------------------

 - [x] Architecture-independent `noarch`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

 - [x] Architecture-independent `noarch`

### CODE REVIEW

File: extra-i18n/fcitx5-material-color/autobuild/build [hunk 1]
```diff
@@ -1,11 +1,19 @@
 abinfo "Installing file to $PKGDIR..."
 
-install -Dm644 arrow.png radio.png -t "$PKGDIR"/usr/share/$PKGNAME/
-for _variant in blue brown deepPurple indigo pink red teal; do
-_variant_name=Material-Color-${_variant^}
-    install -Dm644 panel-$_variant.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/panel.png
-    ln -s ../../../$PKGNAME/arrow.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/
-    ln -s ../../../$PKGNAME/radio.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/
-    install -Dm644 theme-${_variant}.conf  "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/theme.conf
+install -Dvm644 arrow.png radio.png -t "$PKGDIR"/usr/share/$PKGNAME/
```
Comment (MingcongBai) [inline extra-i18n/fcitx5-material-color/autobuild/build]: Absolute path here.

File: extra-i18n/fcitx5-material-color/autobuild/build [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+variants=""
+
+for i in `ls panel-*`; do
+    variants+=$(echo "$i" | cut -d '-' -f2 | cut -d '.' -f1)
+    variants+=' '
+done
+
+for _variant in $variants; do
+    _variant_name=Material-Color-${_variant^}
+    install -Dvm644 panel-$_variant.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/panel.png
```
Comment (MingcongBai) [inline extra-i18n/fcitx5-material-color/autobuild/build]: Absolute path here.

File: extra-i18n/fcitx5-material-color/autobuild/build [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+    ln -sv "$SRCDIR"/arrow.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/
+    ln -sv "#SRCDIR"/radio.png "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/
+    install -Dvm644 theme-${_variant}.conf  "$PKGDIR"/usr/share/fcitx5/themes/$_variant_name/theme.conf
```
Comment (MingcongBai) [inline extra-i18n/fcitx5-material-color/autobuild/build]: Absolute path here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by OriginCode** [comment]
LGTM on amd64 (Unable to take a screenshot sry)

**Review by OriginCode — APPROVED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**